engine/pom.xml

```diff
@@ -23,7 +23,7 @@
     <distributionManagement>
         <site>
             <id>site</id>
-            <url>http://validator.hibernate.org</url>
+            <url>http://hibernate.org/validator</url>
```

Comment:
because the first one redirects to the second, that means the second one is your canonical representation.
Also, consistency: there is no such thing as "orm.hibernate.org", so essentially those having a subdomain are legacy forwarding to the new order.